newdle/client/src/components/creation/timeslots/Timeline.module.scss

```diff
@@ -255,6 +255,19 @@ $label-width: 180px;
       padding-bottom: 0.5em;
     }
   }
+
+  :global(.ui.dropdown) {
+    width: 14em;
+
+    :global(.divider.text) {
+      white-space: nowrap;
+      overflow-x: hidden;
```

Comment:
For some reason I get tiny scroll bars as soon as I switch to Firefox's responsive view... Not sure if this happens on actual mobile devices as well or if it's just some Firefox weirdness...

![image](https://user-images.githubusercontent.com/179599/93918552-d1d42e80-fd0c-11ea-9751-44a628ae379c.png)

If I use `overflow: hidden` this does not happen - and I don't think we have any y-overflows anyway where this would be a problem.

```suggestion
      overflow: hidden;
```